docs/ipac/irsa/most/most.rst

```diff
@@ -0,0 +1,449 @@
+
+As demonstrated, the returned values are stored in a dictionary and which
+``metadata`` key table matches the ``Brief`` output mode table.
+
+The ``fits_tarball`` and ``region_tarballs`` keys store the URL of the TAR
+archive containing all 10 images that observed asteroid Victoria on that night.
+Individual images that were put into the archive are stored under the ``results``
+key:
+
+.. doctest-remote-data::
+
+    >>> matched["fits_tarball"]
+    'https://irsa.ipac.caltech.edu/workspace/TMP_X69utS_13312/Most/pid15792/fitsimage_A850RA.tar.gz'
+    >>> matched["region_tarball"]
+    'https://irsa.ipac.caltech.edu/workspace/TMP_X69utS_13312/Most/pid15792/ds9region_A850RA.tar'
+    >>> matched["results"].columns
+    <TableColumns names=('Image_ID','date_obs','time_obs','mjd_obs','ra_obj','dec_obj','sun_dist','geo_dist','dist_ctr','phase','vmag','image_url','postcard_url','region_file')>  # noqa: E501
+    >>> matched["results"]["time_obs", "image_url"]
+    <Table length=10>
+      time_obs                                                  image_url
+       str12                                                      str103
+    ------------ -------------------------------------------------------------------------------------------------------
+    11:00:08.319 https://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/3b/49273b/134/49273b134-w2-int-1b.fits
+    11:00:08.319 https://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/3b/49273b/134/49273b134-w1-int-1b.fits
+    14:09:44.351 https://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/7b/49277b/135/49277b135-w1-int-1b.fits
+    14:09:44.351 https://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/7b/49277b/135/49277b135-w2-int-1b.fits
+    17:19:09.391 https://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/1b/49281b/134/49281b134-w2-int-1b.fits
+    17:19:09.391 https://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/1b/49281b/134/49281b134-w1-int-1b.fits
+    20:28:45.431 https://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/5b/49285b/135/49285b135-w2-int-1b.fits
+    20:28:45.431 https://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/5b/49285b/135/49285b135-w1-int-1b.fits
+    23:38:10.476 https://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/9b/49289b/134/49289b134-w1-int-1b.fits
+    23:38:10.476 https://irsa.ipac.caltech.edu/ibe/data/wise/merge/merge_p1bm_frm/9b/49289b/134/49289b134-w2-int-1b.fits
+
+.. note::
+    The returned TAR Archives point to the Most service hosted directory which
+    will dissapear after a while, making the URLs return a 404 Not Found Error.
+    The URLs returned by the results table, however, point to the NASA/IPAC
+    Infrared Science Archive, which means that the URLs to the images themselves
+    will remain valid even after the Most URLs expire.
+
+
+Reference/API
+=============
+
+.. automodule:: astroquery.ipac.irsa.most
+   :members:
```

Comment:
This is my suspect for the weird looking docs. We do use automodapi. that provides a much nicer API reference, but given this is pulled into the main IRSA namespace, it's already been API docs-ed there. So I would say, drop the reference here, and we'll figure out whether to link it again from the other namespace, or even include this page in there?

```suggestion

```

Reply:
I think the weirdness is all due to the fact that MOST doesn't behave like a simple extension of the core general catalog query system implemented by the IRSA module, so it's difficult so it's difficult to reconcile these differences if they occupy the same namespace. The use of MOST as a service is not simple nor straight-forward because what it returns changes depending on what the query is. This puts the onus of understanding what happens in the back-end on the user. Some of this is hidden by the verification system, forcing user to be capitalization aware for example, some of it by the subtle different mapping between argument names in the `query_region` method and online form. Certainly including this into IRSA would at least double the length of that page.

Up to you. I have no hard preferences about it but it's a tough question. Removed as requested for now.